src/cmd/new/new.go

```diff
@@ -0,0 +1,33 @@
+package cmdNew
+
+import (
+	_ "embed"
+	"fmt"
+	"github.com/spf13/cobra"
+	"os"
+)
+
+var NewCmd = &cobra.Command{
+	Use:   "new {ruleset name}",
+	Short: "Creates new ruleset",
+	Args:  cobra.ExactArgs(1),
+	Run: func(cmd *cobra.Command, args []string) {
+		err := newRuleset(args[0])
+		if err != nil {
+			fmt.Println("Failed to create new ruleset")
+			fmt.Printf("ERROR: %s\n", err)
+			os.Exit(1)
+		}
+	},
+}
+
+//go:embed "initial_ruleset/spito.yaml"
+var spitoYaml []byte
+
+func newRuleset(name string) error {
+	if err := os.Mkdir(name, os.ModePerm); err != nil {
+		return err
+	}
+
+	return os.WriteFile(name+"/spito.yaml", spitoYaml, os.ModePerm)
```

Comment:
Where exactly will this spito.yaml file be?

Reply:
We discussed in the last meeting that `spito-rules.yaml` file, which you can find in [this](https://github.com/Nasz-Elektryk/spito-ruleset/) repo, will be renamed to `spito.yaml`

Reply:
OK, I understand. I humbly propose to rename this `name` variable something like `directoryName` to avoid confusion.